### ISSUE TITLE
setup

### ISSUE THREAD

**Issue by hisbalah** [hoaan1995/Cosmic-Mirai#24]
every single command works according procedure, but when I put  cd ~/
chmod 777 * 
sh build.sh
                                            there is no file build.sh      and I searched for it.   what is wrong?

**Comment by KaliKacsa** [comment]
> 
HERE: PUT THIS



cd /tmp
wget https://dl.google.com/go/go1.13.linux-amd64.tar.gz
sha256sum go1.13.linux-amd64.tar.gz
sudo tar -C /usr/local -xzf go1.13.linux-amd64.tar.gz
export PATH=$PATH:/usr/local/go/bin
export GOROOT=/usr/local/go
export GOPATH=$HOME/Projects/Proj1
export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
export GOROOT=/usr/local/go; export GOPATH=$HOME/Projects/Proj1; export PATH=$GOPATH/bin:$GOROOT/bin:$PATH; go get github.com/go-sql-driver/mysql; go get github.com/mattn/go-shellwords
source ~/.bash_profile
go version
go env
cd ~/

**Comment by hisbalah** [comment]
if there is no file  build.sh in the folder when i check it with command  -- ls --, how can i start instatation??? no command will help me to allow run file, if there is no file.   of course I put also that command, but i have no idea after which command, is downloaded build.sh  and where.

**Comment by KaliKacsa** [comment]
> if there is no file build.sh in the folder when i check it with command -- ls --, how can i start instatation??? no command will help me to allow run file, if there is no file. of course I put also that command, but i have no idea after which command, is downloaded build.sh and where.

I think you forget to upload all file into your vps

**Comment by hisbalah** [comment]
thank you, I got it, now i have another, proglem :D :D with putty, i do not know this program, and anyway, du i need to put my bank detail on   VULTR page, to be able to run putty? cos i can not run it. hah

**Comment by KaliKacsa** [comment]
> thank you, I got it, now i have another, proglem :D :D with putty, i do not know this program, and anyway, du i need to put my bank detail on VULTR page, to be able to run putty? cos i can not run it. hah



I used 2 program which is MobaXterm and Putty.
I run VPS on DigitalOcean, then i downloaded MobaXterm ( for login to the vps) Putty is for login to the botnet this the easiest method.

**Comment by hisbalah** [comment]
thank you, so it means I have to put my bank details on DigitalOcean, and pay, that is a litle problem, only if i could buy credit card, with credit.  I wont put my bank details.

**Comment by KaliKacsa** [comment]
> thank you, so it means I have to put my bank details on DigitalOcean, and pay, that is a litle problem, only if i could buy credit card, with credit. I wont put my bank details.


It only cost 1$ for register. they have multiple server the cheapest is 4$ I recommend the 6$ the good thing is you only have to pay 6$ if you reach the budget which is  6$. Its cost  1 or 2 cent every hour. Im using almost 3 days and it cost me 14 cent.

Its up to you. 

( I found a  free options i used this for small projekts.  its completly free so you just need to create an account, but its very limited.
if you want to try, try it. )

Link:

https://ide.goorm.io/

**Comment by hisbalah** [comment]
THANK YOU, but I think you are wasting time with me.   look  https://ibb.co/VpxtNp6       I am gonna radther sell vegetable.

**Comment by KaliKacsa** [comment]
> THANK YOU, but I think you are wasting time with me. look https://ibb.co/VpxtNp6 I am gonna radther sell vegetable.

Yeah, i see, this wont work because goorm only support Ubuntu redistribution no other OS. These commands made for CentOs.

**Comment by hisbalah** [comment]
Hey man, i already found solution. I instaled centos on virtualbox, in virtualbox while is turend off, right click, net, settings change from and select Bridged adapter, to be able to connect on IP in Local Network. :D I am IN, now I have to set up everything.  I will let you know later. I have to set up everything. I just wana try it now. 🗡️ :D

**Comment by KaliKacsa** [comment]
> Hey man, i already found solution. I instaled centos on virtualbox, in virtualbox while is turend off, right click, net, settings change from and select Bridged adapter, to be able to connect on IP in Local Network. :D I am IN, now I have to set up everything. I will let you know later. I have to set up everything. I just wana try it now. 🗡️ :D

Well done, this is an option too. 
Good luck! 👍

**Comment by hisbalah** [comment]
so I DID everything, i am connected, but not in bootnet. but like this :D   https://ibb.co/w0VRVd5

**Comment by KaliKacsa** [comment]
> so I DID everything, i am connected, but not in bootnet. but like this :D https://ibb.co/w0VRVd5

Yes, the correct way to load bots is You have a Exploit folder and then there is 3 folder(DVR, fiber, zhone) these are the exploits for the bots. Open them and you'll see a file called tutorial.txt All of the instruction is in there!

But you have to login 3x to your server to load them!

**Comment by quandeptraivkl** [comment]
I have a problem that after running the build.sh file, the CNC file for the screen is lost. Do you have a way to fix it?

**Comment by KaliKacsa** [comment]
> I have a problem that after running the build.sh file, the CNC file for the screen is lost. Do you have a way to fix it?

Im no longer doing this kind of stuff, i recommend you to stop also. Even if you managed to setup everything it wont work because those exploits are out of date and you'll be able to load 15 or less from 3 exploit which is nothing. These botnets are useless now it was good back in the days but nowdays is useless. Use instead ZxcHDDos tool or whatever it called, its available in here Github. You need only a VPS. Dont use Google VPS, DigitalOcean these are detect your stuff and they ban your account. I dont know the rest. If you need help ask.
